ee/codegen/src/generators/nodes/inline-prompt-node.ts

```diff
@@ -142,6 +144,43 @@ export class InlinePromptNode extends BaseSingleFileNode<
       })
     );
 
+    if (this.nodeData.data.execConfig.settings) {
+      const timeout = this.nodeData.data.execConfig.settings.timeout;
+      const streamEnabled =
+        this.nodeData.data.execConfig.settings.streamEnabled;
+      const args = [];
+      if (!isNil(timeout)) {
+        args.push(
+          python.methodArgument({
+            name: "timeout",
+            value: python.TypeInstantiation.float(timeout),
+          })
+        );
+      }
+      if (!isNil(streamEnabled)) {
+        args.push(
+          python.methodArgument({
+            name: "stream_enabled",
+            value: python.TypeInstantiation.bool(streamEnabled),
```

Comment:
[nit] I think we should still skip if it's the default value (in this case, `true`), since the FE may be sending this value during this dual writing period